src/ConjugatePriors.jl

```diff
@@ -2,12 +2,19 @@ module ConjugatePriors
 
 using Statistics
 using LinearAlgebra
+has_cholesky(a::AbstractMatrix) = isa(try_cholesky(a), Cholesky)
+function try_cholesky(a::AbstractMatrix)
+    try cholesky(a)
+    catch e
+        return e
+   end
+end
 
 using PDMats
 using Distributions
 using StatsFuns
 using SpecialFunctions
-
+lgamma(x) = (logabsgamma(x))[1]
```

Comment:
This seems wrong - it is exported by SpecialFunctions but has been deprecated for a long time (https://github.com/JuliaMath/SpecialFunctions.jl/blob/7461b517bcecd28800eef4fe7d4d263ea1ff66a6/src/deprecated.jl#L6). One should rather replace occurrences of `lgamma` with `logabsgamma`.